src/gc/sample/gcenv.h

```diff
@@ -187,5 +186,35 @@ class EEConfig
 
 extern EEConfig * g_pConfig;
 
+#ifdef FEATURE_STANDALONE_GC
```

Comment:
The GCSample should have any plumbing for the standalone GC. It should be kept simple.